cli/src/main/java/com/liferay/blade/cli/BladeCLI.java

```diff
@@ -112,15 +112,37 @@ public BaseArgs getArgs() {
 	}
 
 	public BladeSettings getBladeSettings() throws IOException {
-		final File settingsFile;
+		File settingsBaseDir;
 
 		if (WorkspaceUtil.isWorkspace(this)) {
-			File workspaceDir = WorkspaceUtil.getWorkspaceDir(this);
-
-			settingsFile = new File(workspaceDir, ".blade/settings.properties");
+			settingsBaseDir = WorkspaceUtil.getWorkspaceDir(this);
 		}
 		else {
-			settingsFile = new File(_USER_HOME_DIR, ".blade/settings.properties");
+			settingsBaseDir = _USER_HOME_DIR;
+		}
+
+		File settingsFile = new File(settingsBaseDir, BladeSettings.BLADE_SETTINGS_OLD_STRING);
+
+		if (settingsFile.exists()) {
```

Comment:
The logic here looks weird.  You check for the old location and if it doesn't exist (a new user), then what happens?  You never set the 'settingsFile' back to new location?  Can you check again?